### ISSUE TITLE
Wound and Strain - Count and Display Bars

### ISSUE THREAD

**Issue by JNogaj** [Mezryss/FVTT-Genesys#89]
As of right now there seems to be no consistent way of recording Wounds and Strain above the Threshold - which is a rather important feature of the system considering Healing and fact being at the Threshold does not incapacitate characters, only crossing it does and we then still count it until double the Threshold.

Also what I think would be the most natural way of handling the bars on tokens is that if you have 0 strain and wounds then bar is full. And as characters accumulate it, the bar starts dropping, reaching 0 when at the Threshold. And then obviously stays there as characters accumulate more and more points above the threshold.

There is non-zero chance I am just blind and didn't notice correct options but that is what I caught.

**Comment by Assembling-Kings** [comment]
> As of right now there seems to be no consistent way of recording Wounds and Strain above the Threshold - which is a rather important feature of the system considering Healing and fact being at the Threshold does not incapacitate characters, only crossing it does and we then still count it until double the Threshold.

I'm not sure what you mean by this. You can specify how many wounds/strain your character has by filling the current field.

**Comment by JNogaj** [comment]
Right now there are two ways of handling it. Either by typing in a new value or by using "+X" and "-X". If you try the latter method and it would bring character over the threshold, it will instead reset it to threshold. Rules of the game very much expect a possibility for current values to exceed thresholds so it creates rather.. confusing situation.

**Comment by Assembling-Kings** [comment]
> Right now there are two ways of handling it. Either by typing in a new value or by using "+X" and "-X". If you try the latter method and it would bring character over the threshold, it will instead reset it to threshold. Rules of the game very much expect a possibility for current values to exceed thresholds so it creates rather.. confusing situation.

So to track wounds or strain you can just input the value and it should be fine. On the following screenshot you can see that I have inputted a value above the threshold and it's saved after I open/close the sheet or even the world.
![image](https://github.com/Mezryss/FVTT-Genesys/assets/1728535/bc304a88-4220-484b-b705-b09cfc9f3a44)

I don't know what you mean about the second method. So we're probably misunderstanding each other. Could you elaborate what is the expected behavior and end goal of what you want to accomplish with as much detail as possible and by using screenshots or drawings when adequate.

**Comment by JNogaj** [comment]
![obraz](https://github.com/Mezryss/FVTT-Genesys/assets/43148782/305b20e2-03eb-4ace-a65d-c69d1a167fbe)![obraz](https://github.com/Mezryss/FVTT-Genesys/assets/43148782/66f6047d-5f75-4ffb-b037-db46b395fe4b)

![obraz](https://github.com/Mezryss/FVTT-Genesys/assets/43148782/436d7dc6-e3e1-49ff-82de-d13a3c6ff34f)
![obraz](https://github.com/Mezryss/FVTT-Genesys/assets/43148782/04ab1c93-a854-43da-a82d-437984bcf78e)
![obraz](https://github.com/Mezryss/FVTT-Genesys/assets/43148782/9b8f5c83-8b72-4e2f-a90c-191366d215f8)
![obraz](https://github.com/Mezryss/FVTT-Genesys/assets/43148782/8b18305a-01b6-493f-9332-da5ed8df5a33)

Perhaps this explains better what I mean. It is a small thing but one that potentially leads to bugs and confusion.

**Comment by Assembling-Kings** [comment]
Ahhh, you meant updates through a token on the canvas. Gotcha, thanks for the report.

**Comment by JNogaj** [comment]
Indeed! Most users wont be opening sheets in combat so making sure that works is rather important I would imagine. That and setting default bars to being full when 0 and empty when at Threshold.
